docs/docker.md

```diff
@@ -97,3 +97,8 @@ Adding extra options to pass to the docker daemon:
 ## This string should be exactly as you wish it to appear.
 docker_options: ""
 ```
+
+The proxy settings are forwarded to the Docker daemon by default. Set this to `false` if you don't want Docker's outgoing traffic to use the proxy. Note that this flag will have no effect if the `http_proxy` or `https_proxy` variables are not set.
+```yaml
```

Comment:
markdownlint reports the following error：
```
./docs/docker.md:102 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```yaml"]
./docs/docker.md:104:3 MD047/single-trailing-newline Files should end with a single newline character
```
https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/4764950297